en_US/changes/changes-ce-5.4.md

```diff
@@ -0,0 +1,72 @@
+# Releases
```

Comment:
Shouldn't we add 5.4.1 section here as well?

Reply:
This file and the correspondence ee file will not be used anymore. We have put all the previous release history in one file, as required by CEO.